fitbenchmarking/fitbenchmarking
# Remove minus from NIST and Cutest parser

#### Description of Work
This PR removes the minus sign from the NIST and Cutest parsers so that they now return the jacobian of the function and not the (partial) jacobian of the residual.

Fixes #956

#### Testing Instructions

1. Check the changes look sensible and the Analytic jacobian still works.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
